### DISCUSSION TITLE
How to transcribe Chinese audio to text in Simplified Chinese instead of  Traditional Chinese?

### DISCUSSION THREAD

**Discussion by playgithub** [ggml-org/whisper.cpp#2318] [Q&A] [answered]
```sh
./main -l zh -m "/path/to/ggml-medium-q5_0.bin" "/path/to/song.wav"
```

The wav in Chinese is transcribed to text in Traditional Chinese.
Simplified Chinese is in demand, how?

**Comment by playgithub** [comment, marked as answer]
add a praram as below
```
--prompt "以下是普通话的句子。"
```

**Comment by ivanooi** [comment]
this does not work in Windows. Any ideas? Thanks

**Comment by iTula** [comment]
Try to use prompt in SC.
发自我的 iPad

在 2025年6月26日，14:53，ivanooi ***@***.***> 写道：

﻿

this does not work in Windows. Any ideas? Thanks

**Reply by ivanooi** [reply]
SC?

**Reply by ivanooi** [reply]
do you mean in source code? I did that. I'm using Java ProcessBuilder. Thanks

**Comment by iTula** [comment]
Simple Chinese.

2025年6月26日 20:05，ivanooi ***@***.***> 写道：



SC?

**Reply by ivanooi** [reply]
ooo but i need it in traditional chinese ..that why...
